### DISCUSSION TITLE
Component Documentation e.g Storybook

### DISCUSSION THREAD

**Discussion by mattgreenfield** [livewire/livewire#3673] [1. Help]
Hi,

I'm wondering if there's a 'go to' library for documenting Livewire and blade components?

I've used Storybook in the past for React and Vue projects and wondered if there is something similar in PHP land?

Storybook does support plain HTML components so I've been wondering about some kind of Node based Livewire to HTML option too.

Thanks in advance,
Matt

**Comment by joshhanley** [comment]
@mattgreenfield none that I know of. But I'd be interested to know if you do find something 🙂

**Comment by fuel-wlightning** [comment]
I was wondering about this myself, I haven't seen anything specific to Livewire, but I did see this which can render laravel blades within Storybook, which you might be able to embed a Livewire component in?

https://github.com/area17/blast

Hope this helps! I'm not yet up to doing this within my own applications, but have filed it away for when I'm ready.
(Note: the repo above has a commit as of 13 days ago at the writing of this, so seems actively developed)

**Reply by mattgreenfield** [reply]
Thank you, this looks great. I will give it a go when I find the time.....

**Reply by pxlrbt** [reply]
Did anybody test this with Livewire components?

**Comment by Sophist-UK** [comment]
Definitely interested in this.

**Reply by pxlrbt** [reply]
Try Blast. It should work with some setup. Had a PoC for Filament: https://github.com/area17/blast

**Comment by Sophist-UK** [comment]
Yes - I have already found Blast.
